server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/controller/ComparisonAuditControllerTests.java

```diff
@@ -100,7 +100,7 @@ public void IRVContestMakesIRVAudit() {
     testUtils.log(LOGGER, "IRVContestMakesIRVAudit");
 
```

Comment:
Again, I would add a little comment before the first use of ContestCounter.countAllContests(true) to explain the flag. And say that it applies to subsequent uses.